packages/portfolio-deploy/test/portfolio-start.core.test.ts

```diff
@@ -97,9 +102,9 @@ test('coreEval code without swingset', async t => {
   );
 
   const options = toExternalConfig(
```

Comment:
If we namespace the options by nesting in a `portfolio` property, all `options = toExternalConfig(` will need to look like `options = { portfolio: toExternalConfig(`.